azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +30,8 @@
   SUCCEEDED(50),
   KILLING(55),
   KILLED(60),
+  // POD_FAILED refers to a failed containerized flow due to pod failure
+  POD_FAILED(65),
```

Comment:
We should consider giving this state a more generic name which could be used for flows in similar state in non-containerized Azkban. For instance in the event of a bare-metal executor crashing we could the switch any flows assigned to that executor to this state. 
Something like `EXECUTE_INFRA_FAILED`, better alternatives should be possible though.

Reply:
+1 on using a generic name not indicative or coupled to a specific implementation.